benchmarks/performance-poetry/complex-benchmark/src/main/java/com/squareup/benchmarks/performance/poetry/complex/benchmark/ComplexPoetryResults.txt

```diff
@@ -0,0 +1,106 @@
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-20Ms   min 0.2,   median 0.2,   max 0.8
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-21Ms   min 0.2,   median 0.3,   max 0.4
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-22Ms   min 0.1,   median 0.2,   max 0.2
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-23Ms   min 0.1,   median 0.3,   max 0.6
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-24Ms   min 0.1,   median 0.2,   max 0.3
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-25Ms   min 0.3,   median 0.4,   max 0.4
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-26Ms   min 0.1,   median 0.2,   max 0.2
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-27Ms   min 0.5,   median 0.6,   max 0.6
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-28Ms   min 0.2,   median 0.2,   max 0.4
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-29Ms   min 0.3,   median 0.3,   max 1.0
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-2Ms   min 0.0,   median 0.0,   max 0.1
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-30Ms   min 0.1,   median 0.2,   max 0.3
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-31Ms   min 0.2,   median 0.6,   max 0.7
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-3Ms   min 0.0,   median 0.0,   max 0.0
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-4Ms   min 0.2,   median 0.6,   max 0.9
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-5Ms   min 0.1,   median 0.2,   max 0.4
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-6Ms   min 0.0,   median 0.1,   max 0.2
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-7Ms   min 0.1,   median 0.2,   max 0.2
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-8Ms   min 0.0,   median 0.1,   max 0.1
+WorkflowIdentifier(com.squareup.benchmarks.performance.complex.poetry.PerformancePoemsBrowserWorkflow)-9Ms   min 0.3,   median 0.4,   max 1.0
+Traces: Iteration 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19
```

Comment:
What this log `Traces: Iteration 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19` telling us exactly?

Reply:
I just realized this might not be a file you created, but instead generated by macrobenchmark?

Reply:
These are the test results generated by macrobenchmark. I copied them here to save them. This just shows we did 20 iterations (in the results this is also a link to the trace file of each iteration).